packages/framework/src/Model/Mail/MailTemplateFacade.php

```diff
@@ -115,4 +110,16 @@ public function existsTemplateWithEnabledSendingHavingEmptyBodyOrSubject()
     {
         return $this->mailTemplateRepository->existsTemplateWithEnabledSendingHavingEmptyBodyOrSubject();
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Mail\MailTemplate $mailTemplate
+     * @return \Shopsys\FrameworkBundle\Model\Mail\MailTemplate
+     */
+    public function getTemplateWrappedWithGrapesBody(MailTemplate $mailTemplate): MailTemplate
```

Comment:
I would like to talk about the necessity of the two methods and why it's not working by default :)

Reply:
Basically, we do not want to store the wrapper code in the DB for each template as it would make it difficult to update. So, the wrapped one is necessary for all the methods that then send the template while the one without the wrapper (regular `getById()`) is necessary for the admin edit page of the given mail template - the plain mail template body is then wrapped thanks to `InitGrapesJs` and `grapesJsMailType.html.twig` where a Twig extension is used - https://github.com/shopsys/shopsys/blob/16.0/packages/framework/src/Twig/GrapesJsMailExtension.php#L38. Thanks to that, we avoid the unwanted storing of the wrapped content. It seems like there is room for improvement here, however, I do not have the ambition to rewrite it now TBH :sweat_smile: